sthttpd-cve-2017-10671/mayhem/sthttpd/Mayhemfile

```diff
@@ -0,0 +1,11 @@
+version: '1.4'
+project: sthttpd-cve-2017-10671
+target: sthttpd
+baseimage: forallsecure/sthttpd-cve-2017-10671
+duration: 3600
```

Comment:
Please see what happens with advanced_triage.  This is a feature unique to Mayhem, and should help find more issues.

Reply:
done